src/SprykerEco/Zed/Braintree/Business/BraintreeFacadeInterface.php

```diff
@@ -233,4 +241,37 @@ public function postSaveHook(QuoteTransfer $quoteTransfer, CheckoutResponseTrans
      * @return \Generated\Shared\Transfer\PaymentMethodsTransfer
      */
     public function filterPaypalExpressPaymentMethod(PaymentMethodsTransfer $paymentMethodsTransfer, QuoteTransfer $quoteTransfer): PaymentMethodsTransfer;
+
+    /**
+     * Specification:
+     * - Checks if Braintree has a nonce.
+     * - QuoteTransfer.Payment and QuoteTransfer.Payment.Braintree must be set.
```

Comment:
* Requires `QuoteTransfer.payment` and `QuoteTransfer.payment.braintree` must be set.